scw/client.go

```diff
@@ -90,6 +91,10 @@ func defaultOptions() []ClientOption {
 	return []ClientOption{
 		WithAPIURL("https://api.scaleway.com"),
 		WithUserAgent(userAgent),
+		WithPaginationConfig(&PaginationConfig{
```

Comment:
IMO clientOption must have static parameters:

```Go
WithPaginationConfig(full bool, pageSize uint)
```

Let's decide fully decide what this client option is about so we fix these parameters for good.